cmd/tailscale/cli/up.go

```diff
@@ -580,6 +580,9 @@ func checkForAccidentalSettingReverts(flagSet *flag.FlagSet, curPrefs, newPrefs
 	var missing []string
 	for flagName := range flagsCur {
 		valCur, valNew := flagsCur[flagName], flagsNew[flagName]
+		if flagName == "login-server" && isLoginServerSynonym(valCur) && isLoginServerSynonym(valNew) {
```

Comment:
move this down to line 592 after the existing two checks? seems like the general/common cases should go first.